Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/MotorMachineInterface.mo

```diff
@@ -0,0 +1,106 @@
+  s = if noEvent(omega_s>0) then 1-omega_r/omega_s else 0;
+  ratio = X_m/(X_m+X_s);
+  tau_e = if noEvent(omega_s>0) then n*R_r*s*(V_rms*ratio)^2/(omega_s*(((s*R_s*ratio^2+R_r)^2)+s^2*(X_s+X_r)^2)) else 0;
+
+  annotation (defaultComponentName="torSpe",
+  Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+          Rectangle(
+          extent={{-100,100},{100,-100}},
+          lineColor={0,0,127},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid), Text(
+          extent={{-82,162},{82,116}},
+          lineColor={0,0,255},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.None,
+          textString="%name")}),                                 Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(info="<html>
+<p>This block is to implement equations of the simplified model of squirrel cage type 
+induction motor. The induction motor operation is based on the principle of electromagnetic induction; 
+there are two magnetic fields occurring, one from the stator and one from the rotor, 
+the rotor field is induced by the stator field. Therefore, an induction motor can 
+be modeled as a rotating transformer. </p>
+<p>The per-phase equivalent circuit model for an induction motor supplied by 
+balanced three-phase supply based on the transformer model is represented below. </p>
+<p align=\"center\"><img src=\"modelica://Buildings/Resources/Images/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/induction_motor_equivalent_circuit.png\"/> </p>
+<p>The electromagnetic torque generated by motor is calculated as follows. </p>
+<p align=\"center\"><img src=\"modelica://Buildings/Resources/Images/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/torque_euqation.png\"/> </p>
+Where, <br>
+<<i>n</i> = Number of phases in connected power system <br>
+<i>R<sub>1</i></sub> = Stator resistance [ohm]<br>
+<i>R<sub>2</i></sub> = Rotor resistance [ohm]<br>
+<i>s</i> = Motor slip of asynchronous motors<br>
+<i>V<sub>rms</i></sub> = Root Mean Square (RMS) Voltage of AC waveform [V]<br>
+<i>w<sub>s</i></sub> = Synchronous speed [rad/s] <br>
+<i>X<sub>1</i></sub> = Stator reactance [ohm]<br>
+<i>X<sub>2</i></sub> = Rotor reactance [ohm]<br>
+<i>X<sub>m</i></sub> = Magnetizing reactance [ohm] </p>
+<h4>Assumption and limitation</h4>
+<p>This implementation assumes that the power supply is a balanced system, 
+which allows the total torque to be calculated by multiplying the value of the 
+single-phase torque by the number of phases. </p>
+<h4>Reference</h4>
+<p><a href=\"https://par.nsf.gov/biblio/10109101\">Fu, Yangyang, et al. 
+&quot;Coupling power system dynamics and building dynamics to enabling building-to-grid 
+integration.&quot; <i>Link&ouml;ping Electronic Conference Proceedings</i>. Vol. 157. 2019.</a></p>
+</html>",
+revisions="<html>
+<ul>
+<li>6 March 2019, by Yangyang Fu:<br>First implementation.</li>
```

Comment:
check the format of writing the release note.

Reply:
Thank you for viewing, I've corrected the formatting of the release notes and other sections to make sure they match the other models in MBL.